### PR TITLE
Update data retention instructions to use policy scheduler

### PR DESCRIPTION
**Pull request by mfreed**

### CODE REVIEW

File: using-timescaledb/data-retention.md [hunk 1]
```diff
@@ -20,16 +20,38 @@ the `conditions` hypertable will still have data stretching back 36 hours.
...
+```sql
+SELECT remove_drop_chunks_policy('conditions');
+```
+
+The scheduler framework also allows one to view scheduled jobs:
+```sql
+SELECT * FROM timescaledb_information.drop_chunks_policies;
+```
+
+For more information, please see the [API documentation](add_drop_chunks_policy).
```
Comment (solugebefola) [inline using-timescaledb/data-retention.md]: I think you need square brackets for the link name here.

File: using-timescaledb/data-retention.md [hunk 2]
```diff
@@ -20,16 +20,38 @@ the `conditions` hypertable will still have data stretching back 36 hours.
...
-implement such policies.
+TimescaleDB includes a background job scheduling framework for automating data
+management tasks, such as enabling easy data retention policies.
 
-Note that the path to `psql` and the parameters to connect to the
-database might differ depending on setup and need to be modified
-accordingly in the examples below.
+To add such data retention policies, a database administrator can create,
+remove, or alter policies that schedule `drop_chunks` to take place on a
+hypertable on a regularly scheduled interval.
```
Comment (solugebefola) [inline using-timescaledb/data-retention.md]: The wording of this is confusing for the command below.  The reader may question whether the interval parameter in the command is for when the `drop_chunks` command runs, or for the data age to apply `drop_chunks` to.

**Comment by solugebefola** [comment]
Does this PR cover https://www.pivotaltracker.com/story/show/160502654?

File: using-timescaledb/data-retention.md (hunk 2 above)
Comment (mfreed) [reply using-timescaledb/data-retention.md]: Updated.

File: using-timescaledb/data-retention.md (hunk 1 above)
Comment (mfreed) [reply using-timescaledb/data-retention.md]: Updated.

**Pull request merged**